configs/rhel-program--image-azure.yaml

```diff
@@ -0,0 +1,340 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: RHEL Image Azure
+  description: Packages in the RHEL Image Azure
+  maintainer: bstinson
+  labels:
+    - eln
+    - c10s
+  arch_packages:
+    aarch64:
+      - NetworkManager
+      - NetworkManager-cloud-setup
+      - NetworkManager-tui
+      - PackageKit
+      - WALinuxAgent
+      - acl
+      - amd-gpu-firmware
+      - at
+      - attr
+      - audit
+      - authselect
+      - basesystem
+      - bash
+      - bash-color-prompt
+      - bash-completion
+      - bc
+      - blktrace
+      - bluez
+      - bpftool
+      - bzip2
+      - chrony
+      - cloud-init
+      - cloud-utils-growpart
+      - cockpit
+      - cockpit-bridge
+      - cockpit-packagekit
+      - cockpit-storaged
+      - cockpit-system
+      - cockpit-ws
+      - coreutils
+      - cpio
+      - cronie
+      - crontabs
+      - crypto-policies
+      - crypto-policies-scripts
+      - cryptsetup
+      - curl
+      - cyrus-sasl-plain
+      - dbus
+      - dnf
+      - dnf-plugins-core
+      - dos2unix
+      - dosfstools
+      - dracut-config-generic
+      - e2fsprogs
+      - ed
+      - efibootmgr
+      - ethtool
+      - file
+      - filesystem
+      - glibc
+      - gnupg2
+      - grub2-efi-aa64
+      - grub2-tools
+      - grubby
+      - hostname
+      - hyperv-daemons
+      - initscripts-rename-device
+      - insights-client
+      - intel-gpu-firmware
+      - iproute
+      - iproute-tc
+      - iputils
+      - irqbalance
+      - kbd
+      - kernel
+      - kernel-core
+      - kernel-modules
+      - kernel-tools
+      - kexec-tools
+      - kpatch
+      - kpatch-dnf
```

Comment:
Same here wrt dnf vs. dnf5, but `kpatch-dnf` is a placeholder.

Reply:
There are other placeholders as well: `insights-client` and `rhc`, and possibly more I missed.  Since we have to split the configs anyway, the placeholders can be included in the c10s configs, but not in ELN's.  ELN comps should have all placeholders conditionalized already.